Dockerfile

```diff
@@ -1,7 +1,24 @@
-FROM golang:latest
+# builder
+FROM golang:latest AS builder
 RUN mkdir /app
 ADD . /app/
 WORKDIR /app
+
 RUN go mod download
-RUN go build -o main .
-CMD ["/app/main", "--portHTTP=80"]
\ No newline at end of file
+RUN make build
+RUN ls -l .
+
+# certificates are required in case the Go binary do HTTPS calls
+# to read more about it: https://www.docker.com/blog/docker-golang/ "The special case of SSL certificates"
+FROM alpine:latest as certs
+RUN apk --no-cache add ca-certificates
+
+# runner
+FROM scratch
+#WORKDIR /app
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=builder /app/scribblers /scribblers
+COPY --from=builder /app/templates /templates/
```

Comment:
Uhm, since we invoke `make` which invokes `pkger`, i believe this isn't necessary anymore, as it simply duplicates the templates? They are part of the binary either way if `pkger` was invoked.

Reply:
The files are not part of the binary, as the binary explicitly calls the filesystem to load them.
If you don't add the files, the binary panic with that error message:

```
$ docker run 92b389cabc41
panic: open /templates/error.html: no such file or directory

goroutine 1 [running]:
github.com/scribble-rs/scribble.rs/communication.readTemplateFile(0x87be83, 0xa, 0x0, 0x0)
	/app/communication/init.go:73 +0x1c8
github.com/scribble-rs/scribble.rs/communication.init.0()
	/app/communication/init.go:23 +0x5d
```

Reply:
Hm, hold on. This shouldn't happen. This is exactly what pkger does. pkger represents a virtual file-system that falls back to the real file system if the files aren't packaged.

Reply:
Ok found the issue: templates are not loaded with `pkger`:
```go
errorPage, parseError = template.New("error.html").Parse(readTemplateFile("error.html"))
```
Compared to templates:
```go
templateHandle, pkgerError := pkger.Open("/templates/" + name)
```

Reply:
Appears that smth is wrong there. I'll look into it.

Reply:
Do you believe this should be fixed in this MR?
Or the fix coming in will update the Dockerfile?

Reply:
I am not sure what the problem is yet, so I don't know. The documentation on pkger is either not sufficient enough, there's a bug or I am stupid. One of those three, not sure which one it is yet :D

Reply:
Well, I created an issue. Let's just wait til we resolve this: https://github.com/markbates/pkger/issues/86

If nobody replies, we'll apply the workaround I presented. Alternatively we don't really need to execute packager but can simply copy the resources and templates folder into the docker container, right?

Reply:
Will add a workaround comment for the Dockerfile line, with link to the issue.
And we can move forward with this MR? Will like to take a look at this missbahvior as well.

Reply:
Comment added.